irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -26,28 +27,26 @@ namespace iroha {
     using network::PeerCommunicationService;
 
     TransactionProcessorImpl::TransactionProcessorImpl(
-        PeerCommunicationService &pcs,
-        const StatelessValidator &validator)
-        : pcs_(pcs),
-          validator_(validator) {
-    }
+        PeerCommunicationService &pcs, const StatelessValidator &validator)
+        : pcs_(pcs), validator_(validator) {}
 
-    void TransactionProcessorImpl::transaction_handle(model::Transaction &transaction) {
+    void TransactionProcessorImpl::transactionHandle(
+        std::shared_ptr<model::Transaction> transaction) {
       model::TransactionStatelessResponse response;
-      response.transaction = transaction;
+      response.transaction = *transaction;
```

Comment:
maybe use `shared_ptr` for `response.transaction` as well? so don't need to copy